src/uwtools/resources/rocoto.jsonschema

```diff
@@ -3,68 +3,241 @@
+                    },
+                    "oneOf": [
+                        {
+                            "$ref": "#/$defs/taskPattern"
+                        },
+                        {
+                            "$ref": "#/$defs/metataskPattern"
+                        }
+                    ]
                 }
-            },
-            "required": ["attrs", "cycledefs", "log"]
+            }
         }
-    }
+    },
+    "type": "object",
+        "properties": {
+            "workflow": {
+                "type": "object",
+                "properties": {
+                    "attrs": {
+                        "type": "object",
+                        "properties": {
+                            "realtime": {
+                                "type": "boolean"
+                            },
+                            "scheduler": {
+                                "type": "string",
+                                "enum": [
+                                    "sge",
+                                    "lsf",
+                                    "lsfcray",
+                                    "ll",
+                                    "torque",
+                                    "moabtorque",
+                                    "moab",
+                                    "pbspro",
+                                    "slurm",
+                                    "cobalt"
+                                ]
+                            },
+                            "cyclethrottle": {
+                                "type": "integer",
+                                "minimum": 0
+                            },
+                            "taskthrottle": {
+                                "type": "integer",
+                                "minimum": 0
+                            }
+                        },
+                        "required": ["realtime", "scheduler"]
+                    },
+                    "cycledefs": {
+                        "type": "object",
+                        "properties": {
+                            "groupname": {
+                                "type": "array",
+                                "items": {
+                                    "$ref": "#/$defs/dateTimePattern"
+                                }, 
+                                "required": ["ref"]
+                            },
+                            "required": ["items"]
+                        }
+                    },
+                    "log": {
+                        "type": "string",
+                        "format": "uri"
+                    },
+                    "entities": {
+                        "type": "object"
+                    },
+                    "tasks": {
+                        "type": "object",
+                        "$ref": "#/$defs/taskPattern",
+                        "$ref": "#/$defs/metataskPattern"
```

Comment:
Figured out I had this added in that made it a circular pattern, here for the `tasks` section I only needed `taskPattern` and then call `metataskPattern` from `taskPattern`!